lib/github_backend.rb

```diff
@@ -16,6 +16,15 @@ def initialize(_args = {})
     @logger.level = Logger::DEBUG unless ENV['RACK_ENV'] == 'production'
   end
 
+  def testable_repos
+    projects = JSON.parse(Faraday.get('https://team-api.18f.gov/public/api/projects/').body)
```

Comment:
This is the only place the URL is used, so I don't think it warrants creating a constant or variable for it.